core/src/main/scala/io/qbeast/core/model/Block.scala

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
This is not the right license header, I'm wondering how the build is passing with incorrect headers, is it only processing `src/` and not `core/`?

Reply:
Seems so... would add an issue.

Reply:
Added headers on 883bed17